README.md

```diff
@@ -1,3 +1,5 @@
+![Build](https://github.com/vilvo/kafka-connect-snmp/workflows/Build/badge.svg)
```

Comment:
interesting, why do you have a badge as svg and not really pointing to a real build?

Reply:
Github actions badges are indeed referred through workflow badge.svg - see https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository

Reply:
Yeah I meant something like this

```
[![Build](https://github.com/ElisaOyj/kafka-connect-snmp/workflows/Build/badge.svg)](https://github.com/ElisaOyj/kafka-connect-snmp/actions?query=branch%3Amaster)
```